samples/src/main/java/org/ishugaliy/allgood/consistent/hash/samples/metrics/HashRingMetrics.java

```diff
@@ -1,13 +1,19 @@
-package org.ishugaliy.consistent.hash.samples.metrics;
+package org.ishugaliy.allgood.consistent.hash.samples.metrics;
 
-import org.ishugaliy.consistent.hash.ConsistentHash;
-import org.ishugaliy.consistent.hash.HashRing;
-import org.ishugaliy.consistent.hash.node.Node;
+import org.ishugaliy.allgood.consistent.hash.ConsistentHash;
+import org.ishugaliy.allgood.consistent.hash.HashRing;
+import org.ishugaliy.allgood.consistent.hash.node.Node;
 
 import java.util.*;
 
-import static java.lang.String.*;
+import static java.lang.String.format;
 
+/**
+ * Decorator of {@link HashRing} that proxy all api calls to the target hash ring
+ * and calculated load distribution statistic.
```

Comment:
{@link HashRing} decorator.
 Proxies all API calls to the target hash ring and calculates load distribution statistic.